File: 01_simple_applications/03_average.py
# http://progra.usm.cl/apunte/ejercicios/1/promedio.html

labels = ["Primera", "Segunda", "Tercera", "Cuarta"] 
average = 0.0

for label in labels:
  value = input(f"{label} nota: ")
  value = float(value)
  average += value

print("El promedio es: " + str(average / len(labels)))

File: src/pt1/03_average.js
import { prompt } from "../utils/stdin.js";

var nota01 = parseInt(await prompt("Ingrese primera nota: "));
var nota02 = parseInt(await prompt("Ingrese segunda nota: "));
var nota03 = parseInt(await prompt("Ingrese tercera nota: "));
var nota04 = parseInt(await prompt("Ingrese cuarta nota: "));
var suma = nota01 + nota02 + nota03 + nota04;
var promedio = suma / 4;

console.log("El promedio es: " + promedio);
